src/main/java/GameBase.java

```diff
@@ -0,0 +1,41 @@
+import java.util.ArrayList;
+import java.util.Random;
+
+public class GameBase {
+    Random random;
+    ArrayList<Integer> originNumber = new ArrayList<>();
```

Comment:
객체 지향에는 다형성이라는 개념이 있음 
```suggestion
    List<Integer> originNumber = new ArrayList<>();
```

Reply:
선언 이렇게하는게 더 좋은 습관이겠군

Reply:
습관이라기보다 왜 그런지 알아야하는게 좋은데...

Reply:
`originNumbers` 혹은 `originNumberList` 와 같이 리스트임을 네이밍에서도 알수 있도록 하는게 좋을것 같습니다.